Ethnologie française (French)

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Ethnologie française (French)</title>
+    <id>http://www.zotero.org/styles/ethnologie-francaise</id>
+    <link href="http://www.zotero.org/styles/ethnologiefrancaise" rel="self"/>
+    <link href="http://www.puf.com/puf_wiki/images/e/e3/Charte_Ethnologie_fran%C3%A7aise.pdf" rel="documentation"/>
+    <author>
+      <name>Viera Rebolledo-Dhuin</name>
+      <email>viera.rebolledo-dhuin@uvsq.fr</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="history"/>
+    <category field="anthropology"/>
+	<issn>0046-2616</issn>
+    <eissn>2101-0064</eissn>
+    <summary>Author-date style for Ethnologie française - revue des PUR</summary>
+    <updated>2015-11-14T04:18:55+00:00</updated>
+    <rights>This work is licensed under a Creative Commons Attribution-Share Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/</rights>
+  </info>
+ <locale xml:lang="fr">
+    <terms>      
+      <term name="in">in</term>
+      <term name="anonymous">anonyme</term>
+      <term name="anonymous" form="short">anon.</term>
+      <term name="accessed">consulté&#160;le</term>
+      <term name="no date">sans date</term>
+	  <term name="translator" form="short">trad.</term>
+	  <term name="editor" form="short">
+		<single>dir.</single>
+        <multiple>dir.</multiple>
+		</term>
+      <term name="no date" form="short">[s.&#160;d.]</term>	  
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" delimiter=", ">
+      <name and="text" name-as-sort-order="all" sort-separator=" " delimiter=", ">		
+		<name-part name="family" font-variant="small-caps" />
+		<name-part name="given" />
+		</name>
+      <substitute>
+        <names variable="editor"/>
+		<label form="short" text-case="lowercase" prefix=" (" suffix=")"/>
```

Comment:
you'll want to take this line and move it up after line 38. You can't have a label in substitute.